src/components/Tabs.tsx

```diff
@@ -35,38 +39,137 @@ const Tabs = ({ tabs }: TabsProps) => {
     if (idxFromSearch !== selectedIdx) setSelectedIdx(idxFromSearch);
   }, [idxFromSearch, selectedIdx]);
 
+  const changeMenu = (value: string) => {
+    setOpen(false);
+    setSelectedMenu(value);
+  };
+
   return (
-    <div className="w-full max-w-5xl px-2 py-2 sm:px-0 m-auto">
+    <div className="w-full max-w-4xl px-2 py-2 sm:px-0 m-auto">
       <Tab.Group selectedIndex={selectedIdx} onChange={handleChange}>
-        <Tab.List className="flex space-x-1 rounded-xl bg-teal-900/20 p-1">
-          {tabs.map(({ title }) => (
-            <Tab
-              key={title}
-              className={({ selected }) =>
-                classNames(
-                  "w-full rounded-lg py-2.5 text-sm font-medium leading-5 text-teal-700",
-                  "ring-white ring-opacity-60 ring-offset-2 ring-offset-teal-400 focus:outline-none focus:ring-2",
-                  selected
-                    ? "bg-white shadow"
-                    : "text-teal-100 hover:bg-white/[0.12] hover:text-white",
-                )
-              }
-            >
-              {title}
-            </Tab>
-          ))}
-        </Tab.List>
-        <Tab.Panels className="mt-6">
-          {tabs.map(({ content }, idx) => (
-            <Tab.Panel
-              key={idx}
-              unmount={false}
-              className="flex flex-col min-w-full rounded-xl bg-white p-3"
-            >
-              {content}
-            </Tab.Panel>
-          ))}
-        </Tab.Panels>
+        <div className={"flex flex-wrap rounded-xl bg-white"}>
+          <div className={"basis-full md:basis-5/12 md:p-[40px]"}>
```

Comment:
Can we adjust this so more screen width is used? Currently seeing ~896px used on desktop where I'd expect to see ~1200px.

<details>
<summary>screenshot</summary>

<img width="1691" alt="Screenshot 2024-02-28 at 3 06 19 PM" src="https://github.com/DCFoundation/cosmos-proposal-builder/assets/11021913/cdfb1e56-8a30-4e15-9a3d-259a5b963e5c">

</details>